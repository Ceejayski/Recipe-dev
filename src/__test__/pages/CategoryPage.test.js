import { Provider } from 'react-redux';
import { create } from 'react-test-renderer';
import CategoryPage from '../../pages/categoryPage';
import store from '../../redux';

test('renders Correctly', () => {
  const Placeholder = () => (
    <Provider store={store}>
      <CategoryPage />
    </Provider>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
