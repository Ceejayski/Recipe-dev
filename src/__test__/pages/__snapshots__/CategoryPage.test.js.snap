// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Correctly 1`] = `
<div>
  <div
    className="mt-5 container"
  >
    <div
      className="mt-5 d-flex justify-content-between"
    >
      <div
        className="items mt-3"
      >
        <div
          className="category-filter"
        >
          <select
            className="form-select"
            onChange={[Function]}
            value=""
          >
            <option
              value="Pork"
            >
              Pork
            </option>
            <option
              value="Chicken"
            >
              Chicken
            </option>
            <option
              value="Lamb"
            >
              Lamb
            </option>
            <option
              value="Side"
            >
              Side
            </option>
            <option
              value="Miscellaneous"
            >
              Miscellaneous
            </option>
            <option
              value="Goat"
            >
              Goat
            </option>
          </select>
        </div>
      </div>
      <div
        className="items mt-3"
      >
        <h3>
          
        </h3>
      </div>
      <div
        className="items mt-3"
      />
    </div>
    <div
      className="mt-4 row"
    />
  </div>
</div>
`;
